miso-web/src/main/webapp/scripts/bulk.js

```diff
@@ -3362,23 +3362,24 @@ BulkUtils = (function ($) {
 
   function textValidator(column) {
     var sanitize = new RegExp(Utils.validation.sanitizeRegex);
-    var regex = column.regex ? new RegExp(column.regex) : null;
     return function (value, callback) {
       if (Utils.validation.isEmpty(value)) {
         return validateEmpty(this, callback);
       } else if (column.maxLength && value.length > column.maxLength) {
         callback(false);
         return "Max length is " + column.maxLength + " characters";
+      } else if (column.regex) {
```

Comment:
If column.regex exists, then 'sanitize' is never tested?

Reply:
Sanitize is the default and prevents characters that we don't want to allow in most fields. Specifying a regex overrides that. Among other things, this can be used to allow characters that would normally be restricted by sanitize, as in this case. The form pages already work like this, but I guess we never had to allow characters that were prevented by sanitize on the bulk pages before.